### ISSUE TITLE
spyder has error in loading GL command

### ISSUE THREAD

**Issue by Tejaswi007** [spyder-ide/spyder#7599]
the error is as follows:
```
(base) C:\Users\Tejaswi Samavedula>spyder
[4952:5432:0802/115031.311:ERROR:gles2_cmd_decoder.cc(2439)] [GroupMarkerNotSet(
crbug.com/242999)!:C4F5E908]GL ERROR :GL_INVALID_ENUM : BackFramebuffer::Create:
 <- error from previous GL command
[4952:5432:0802/115033.066:ERROR:gles2_cmd_decoder.cc(2439)] [GroupMarkerNotSet(
crbug.com/242999)!:3485EA08]GL ERROR :GL_INVALID_ENUM : BackFramebuffer::Create:
 <- error from previous GL command
[4952:5432:0802/115034.623:ERROR:gles2_cmd_decoder.cc(2439)] [GroupMarkerNotSet(
crbug.com/242999)!:C48EEA08]GL ERROR :GL_INVALID_ENUM : BackFramebuffer::Create:
 <- error from previous GL command
[4952:5432:0802/115034.634:ERROR:gles2_cmd_decoder.cc(2439)] [GroupMarkerNotSet(
crbug.com/242999)!:64EEEA08]GL ERROR :GL_INVALID_ENUM : BackFramebuffer::Create:
 <- error from previous GL command
[5892:5896:0802/115112.099:ERROR:broker_win.cc(59)] Error reading broker pipe: T
he pipe has been ended. (0x6D)
```

Can you give some idea to resolve this error

**Comment by ccordoba12** [comment]
Please open an Anaconda Prompt, run there

    conda install pyopengl

and try again.

**Comment by ccordoba12** [comment]
Great! Thanks for letting us know about it :+1:

**Comment by gflores87** [comment]
I did just that but I keep getting errors:

`[12896:22288:0906/090614.408:ERROR:gles2_cmd_decoder.cc(2439)] [GroupMarkerNotSet(crbug.com/242999)!:4829070800000000]GL ERROR :GL_INVALID_VALUE : BackFramebuffer::Create: <- error from previous GL command<br>[12896:22288:0906/090614.420:ERROR:gles2_cmd_decoder.cc(2439)] [GroupMarkerNotSet(crbug.com/242999)!:7855070800000000]GL ERROR :GL_INVALID_VALUE : BackFramebuffer::Create: <- error from previous GL command<br>[12896:22288:0906/090617.182:ERROR:gles2_cmd_decoder.cc(2439)] [GroupMarkerNotSet(crbug.com/242999)!:68A5B51600000000]GL ERROR :GL_INVALID_VALUE : BackFramebuffer::Create: <- error from previous GL command<br>[12896:22288:0906/090617.194:ERROR:gles2_cmd_decoder.cc(2439)] [GroupMarkerNotSet(crbug.com/242999)!:1804B61600000000]GL ERROR :GL_INVALID_VALUE : BackFramebuffer::Create: <- error from previous GL command<br>[12896:22288:0906/090617.205:ERROR:gles2_cmd_decoder.cc(2439)] [GroupMarkerNotSet(crbug.com/242999)!:680AB61600000000]GL ERROR :GL_INVALID_VALUE : BackFramebuffer::Create: <- error from previous GL command<br>[12896:22288:0906/090617.216:ERROR:gles2_cmd_decoder.cc(2439)] [GroupMarkerNotSet(crbug.com/242999)!:58DDBA2300000000]GL ERROR :GL_INVALID_VALUE : BackFramebuffer::Create: <- error from previous GL command<br>`

**Comment by ccordoba12** [comment]
@gflores87, that could also be an error with your graphics card drivers (Nvidia, Intel or both). Please update them and try again.

**Comment by gflores87** [comment]
@ccordoba12 sadly, graphic card drivers are up to date. Maybe I need to do some heavy duty maintenance with the drivers though

**Comment by ccordoba12** [comment]
This could be fixed in our next version (3.3.2). Would you like to try downloading Spyder with git and test that?

**Comment by gflores87** [comment]
I'm glad to collaborate in any way I can. As long as you provide some instructions, I'll do my part :)

**Comment by ccordoba12** [comment]
It's kind of tricky and I don't have time to guide you, sorry. Let's hope this is fixed in 3.3.2
